GLOSSARY.md

```diff
@@ -0,0 +1,47 @@
+# GitOps Glossary
+
+This glossary accompanies the [GitOps Principles](./PRINCIPLES.md), and other supporting documents in this repository.
+
+- ## Break Glass
+
+    The temporary suspension of GitOps principles, often accomplished by pausing automated _Reconciliation_.
+    While these principles apply to typical operations, it may at times be necessary to temporarily pause reconciliation, for example during incident management activities.
+    In these cases, other modes of operations should be considered (e.g. manual intervention), followed by any necessary updates to the desired state declarations, and finally resuming reconciliation of the system with the updated declarations.
+    Pragmatic exceptions to these guiding principles are expected from time to time during the journey toward a system being fully managed by GitOps.
+
+- ## Continuous
+
+    By "continuous" we adopt the industry standard to mean that _Reconciliation_ continues to happen, not that it must be instantaneous.
+
+- ## Declarative Description
+
+    Describing the desired state or behavior of a system without specifying how that state will be achieved, thereby separating configuration (the desired state) from the implementation (commands, API calls, scripts etc.) that actually achieves the desired state described in the declarative description.
+
+- ## Desired State
+
+    The aggregate of all configuration data for a system form its _Desired State_ which is defined as data sufficient to recreate the system so that instances of the system are behaviourally indistinguishable.
+
+- ## Drift
+
+    When a system's _Actual State_ changes for any reason other than its versioned _Desired State_ declarations having changed, we say that the system has drifted from its _Desired State_.
+
+- ## Reconciliation
+
+    The process of ensuring that the _Actual State_ of a sytem matches its versioned _Desired State_ declarations.
+    Contrary to CIops, any divergence between the two will trigger reconciliation, regardless of where changes occured.
+    Divergence could be due to the actual state unintentionally _Drifting_ from the desired state declarations, or a new desired state declaration version having been changed intentionally.
+
+- ## Software System
+
+    One or more Runtime environments consisting of resources under management.
+    In each Runtime, management Agents to act on resources according to security policies.
```

Comment:
The second clause of this seems incomplete?

Reply:
@RedbackThomson  I agree it reads incomplete as a paragraph, but I thought this could be addressed after RC 1 since it's been like this since February. See https://github.com/open-gitops/documents/pull/15#pullrequestreview-716397234

PS, while this glossary item was initially drafted with line breaks, I believe it was intended to be a list comprising the different parts of a software system as we currently understand it (see https://github.com/gitops-working-group/gitops-working-group/commit/ff540e1552e5a41c03a1fb317d554567777cdfe5). Like this:

> - **Software System:**
>     - One or more Runtime environments consisting of resources under management
>     - In each Runtime, management Agents to act on resources according to security policies
>     - One or more software Repositories for storing deployable artifacts that may be loaded into the runtime environments, eg. configuration files, code, binaries and packages
>     - One or more Administrators who are responsible for operating the runtime environments ie. installing, starting, stopping and updating software, code, configuration, etc
>     - A set of policies controlling access and management of repos, deployment, runtimes

Now that I say this, it would be an easy fix. Does this definition read more clearly as a list?

Reply:
It does read more clearly as a list, although I was more specifically pointing out the following line:
> In each Runtime, management Agents to act on resources according to security policies

However, I understand if you want to leave the glossary as it stands for another time.

Reply:
@RedbackThomson ok we can change to a list, since that's what it should have been all along. About that one list item, does this feel more clear/complete below?

```diff
- In each Runtime, management Agents to act on resources according to security policies
+ In each Runtime, the management Agents which act on resources according to security policies
```

Reply:
Yes :) That sounds good. Thanks

Reply:
How does this part look now?

Reply:
Looks great, Scott. Thank you